lib/gcpspanner/spanneradapters/backend.go

```diff
@@ -318,9 +331,13 @@ func (s *Backend) convertFeatureResult(featureResult *gcpspanner.FeatureResult)
 			featureResult.LowDate,
 			featureResult.HighDate,
 		),
-		Wpt:                    nil,
-		Spec:                   nil,
-		Usage:                  nil,
+		Wpt:  nil,
+		Spec: nil,
+		Usage: &struct {
+			Chromium *backend.ChromiumUsageInfo "json:\"chromium,omitempty\""
```

Comment:
@jcscottiii Due to the way this is defined in OpenAPI, it seemed like I needed to define the struct like this for each instance that I created a reference to Usage. This could be the wrong approach altogether (let me know if you have a better way), or maybe this should be more explicitly defined in the YAML file.

Reply:
Instead of doing this dynamic struct, I would go with the latter option. Could you modify the openapi YAML so that Usage has its own component?